src/main/java/com/alkemy/ong/controller/CategoryController.java

```diff
@@ -1,14 +1,15 @@
 package com.alkemy.ong.controller;
 
 import com.alkemy.ong.dto.CategoryDto;
-import com.alkemy.ong.service.CategoryService;
+import com.alkemy.ong.service.impl.CategoryService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
+
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/category")
```

Comment:
I suggest only use /categories in stead of /category/categories I think in the tickets only mentions /categories

Reply:
the requestMapping(/category) corresponds to Dante's PR, just use the same route to avoid creating a new conflict, comment out your PR so I can fix it